docs/configuration/cosmos-conf.rst

```diff
@@ -94,6 +94,27 @@ This page lists all available Airflow configurations that affect ``astronomer-co
     - Default: ``profile``
     - Environment Variable: ``AIRFLOW__COSMOS__PROFILE_CACHE_DIR_NAME``
 
+.. _remote_cache_path:
+
+`remote_cache_path`_:
+    The remote path to store the DBT cache. Starting with Cosmos 1.6.0, you can store the dbt ls output as cache in a
```

Comment:
```suggestion
    The remote path to store the `dbt ls` cache. Starting with Cosmos 1.6.0, you can store the dbt ls output as cache in a
```

Reply:
Since we would like to have a single cache path and not for `dbt ls` alone had mentioned only `dbt`. And added the second part because at the moment we would only support `dbt ls` in the cache.